### PR TITLE
JBPM-8529: Designer - Migration path from the legacy process designer

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @romartin @hasys would you mind review this PR?

Depends on:
https://github.com/kiegroup/appformer/pull/708
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/962
https://github.com/kiegroup/kie-wb-common/pull/2664
https://github.com/kiegroup/jbpm-designer/pull/851

**Comment by wmedvede** [comment]
jenkins retest this

File: jbpm-wb-showcase/pom.xml [hunk 1]
```diff
@@ -1587,6 +1587,27 @@
...
+
+    <dependency>
+      <groupId>org.kie.workbench.stunner</groupId>
+      <artifactId>kie-wb-common-stunner-jbpm-designer-integration-client</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kie.workbench.stunner</groupId>
+      <artifactId>kie-wb-common-stunner-jbpm-designer-integration-backend</artifactId>
```
Comment (cristianonicolai) [inline jbpm-wb-showcase/pom.xml]: @wmedvede can you specify this one as runtime scope only?

Comment (wmedvede) [reply jbpm-wb-showcase/pom.xml]: hi @cristianonicolai , fixed, if there are no more comments please approve so we can move this forward as soon we get the greens.

Thanks!

**Comment by wmedvede** [comment]
jenkins execute full downstream build

**Pull request merged**